Makefile

```diff
@@ -39,11 +42,15 @@ requirements.txt: poetry.lock
 
 # linting is flake8
 lint: develop
-	$(POETRY) run flake8 globus_automate_client
+	$(POETRY) run isort --check $(LINT_PATHS)
+	$(POETRY) run black --check $(LINT_PATHS)
```

Comment:
We're probably better off defining the lint paths in pyproject.toml and letting the tools read their own configuration when we run them. It at least removes the need for the extra variable here and keeps the project config slightly more centralized.